### ISSUE TITLE
try pma-voice

### ISSUE THREAD

**Issue by minipunch** [minipunch/usarealismrp-fxserver#2051]
`lb-phone` authors say `pma-voice` works better than `mumble-voip` (which we use atm)

and I think pma-voice has more support atm than mumble-voip does so it is probably worth a shot to switch to it

**Comment by minipunch** [comment]
https://github.com/AvarianKnight/pma-voice

**Comment by Savage912-Official** [comment]
> `lb-phone` authors say `pma-voice` works better than `mumble-voip` (which we use atm)
> 
> and I think pma-voice has more support atm than mumble-voip does so it is probably worth a shot to switch to it

Yeah from experience pms seems to be better and more functionality
